app/uk/gov/hmrc/bindingtariffclassification/controllers/MicroserviceHelloWorld.scala

```diff
@@ -84,7 +85,8 @@ class MicroserviceHelloWorld @Inject()(caseService: CaseService, eventService: E
     Logger.debug(s"Liability Order document inserted: $r3")
 
     // GET ALL
-    val r = Await.result(caseService.get(), 2.seconds)
+
+    val r = Await.result(caseService.get(CaseParamsFilter(), Some("")), 2.seconds)
```

Comment:
Why `Some("")` ?
It is not doing any sorting now...
I suggest to put `None`, or a real field name (but in this case you will need to implement the sorting)

Reply:
Do we even need this controller any more?